app/views/share/_embed_meta.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= ts("Chapters: ") %><%= work.chapter_total_display %><br />
+<%= ts("Fandom: ") %><%= work.tag_groups["Fandom"].map { |fandom| link_to fandom.name, tag_url(fandom) }.join(', ').html_safe %><br />
+<% [Rating, Warning, Relationship, Character, Freeform].each do |klass| %>
+<% if (tags = work.tag_groups[klass.to_s]).present? %>
+<%= ts("#{klass.label_name}: ") %><%= tags.map { |tag| tag.display_name }.join(', ') %><br />
+<% end %>
+<% end %>
+<% if work.series.exists? %>
+<%= ts("Series: ") %><%= series_list_for_feeds(work) %><br />
+<% end %>
+<% if work.summary.present? %>
+<%= ts("Summary: ") %><%= sanitize_field(work, :summary) %>
```

Comment:
I'm also getting escaped HTML for the summary when I pull this branch and try it on webdev or local: `Summary: &lt;p&gt;Quare conare, quaeso. Quid est igitur, cur ita semper deum appellet Epicurus beatum et aeternum?&lt;/p&gt;`